avocado_vt/options.py

```diff
@@ -89,13 +89,14 @@ def __init__(self, options):
         set_opt(self.options, 'vt_smp',
                 settings.get_value('vt.qemu', 'smp', default='2'))
         set_opt(self.options, 'vt_image_type',
-                settings.get_value('vt.qemu', 'image_type', default='qcow2'))
+                settings.get_value('vt.qemu', 'image_type',
+                                   default=SUPPORTED_IMAGE_TYPES[0]))
```

Comment:
Right, it makes sense indeed.  But I guess you mean *introducing* `SUPPORTED_ACCEL_TYPES`, because there isn't one yet.  And on that topic, I'm not sure about the correct supported list.  For example, I have not found *any* usage of the `hax` accelerator in Avocado-VT, tp-qemu or tp-libvirt.

So, I've opened an issue to properly handled this: https://github.com/avocado-framework/avocado-vt/issues/2707

I'm marking this as resolved so that we can move the discussion there.